operations/helm/charts/grafana-agent/CHANGELOG.md

```diff
@@ -14,6 +14,14 @@ Unreleased
 
 - Update RBAC rules to permit `node/metrics`. (@yurii-kryvosheia)
 
+0.13.0 (2023-05-01)
+-------------------
+
+### Enhancements
+
+- Update Grafana Agent version to v0.33.1. (@spartan0x117)
```

Comment:
The unreleased section is being brought in too, can we move those entries down to 0.13.0?